ReactNativeClient/lib/components/screens/note.js

```diff
@@ -599,6 +604,7 @@ class NoteScreenComponent extends BaseScreenComponent {
 			const url = Note.geolocationUrl(note);
 			Linking.openURL(url);
 		} catch (error) {
+			this.close_side_menu();
```

Comment:
No need to create a function for, just put the dispatch call directly here.

Reply:
In the file, there were functions defined for such utility processes, even if the code is for just one line. That's why I thought of making it a function, I will change it and also in this file the component is connected to redux using connect, so if I have to call an action from a function, I will have to connect it with mapDispatchToProps, so can I do that? Or if there is any other way I am missing, please tell me. Anywhere else in this file, they have achieved this by making it a new function.

Reply:
I did it because, at most of the places in the file, there were separate functions for these utility tasks even if there is only one line of code. But as you suggested, I've changed the function to a dispatch call.  Making a new PR.